src/views/DefineCompany.vue

```diff
@@ -2,7 +2,7 @@
   <div>
     <div class="mt-10 company-statement">
       <p>
-        <span class="company-statement-label">{{ getCompanyResources.title }}:</span>
+        <span class="company-statement-label">{{ getCompanyResources.title }}</span>
```

Comment:
I think this might display "null" on the page. You might need to add a v-if for title.

Reply:
It doesn't display anything when `null`, tested!

Reply:
OK, that's good; I just learned something. (Not all frameworks are like this.)

Wouldn't it still be better to not render the span if there's no title?

Reply:
I'm not sure, is there a downside to displaying the span if it is empty?

Reply:
🤷

Reply:
Less (code ) is more right? 😆

Reply:
Added safety check :)